### ISSUE TITLE
mrpt_graphslam_2d invalid isntall rule

### ISSUE THREAD

**Issue by tfoote** [mrpt-ros-pkg/mrpt_slam#54]
From: http://build.ros.org/view/Kbin_uX64/job/Kbin_uX64__mrpt_graphslam_2d__ubuntu_xenial_amd64__binary/322/console

```
-- Installing: /tmp/binarydeb/ros-kinetic-mrpt-graphslam-2d-0.1.6/debian/ros-kinetic-mrpt-graphslam-2d/opt/ros/kinetic/share/mrpt_graphslam_2d/launch/sr_graphslam_demo_gt.launch
CMake Error at cmake_install.cmake:131 (file):
  file INSTALL cannot find
  "/tmp/binarydeb/ros-kinetic-mrpt-graphslam-2d-0.1.6/demo".


make[2]: *** [install] Error 1
Makefile:108: recipe for target 'install' failed
make[2]: Leaving directory '/tmp/binarydeb/ros-kinetic-mrpt-graphslam-2d-0.1.6/obj-x86_64-linux-gnu'
	cd /tmp/binarydeb/ros-kinetic-mrpt-graphslam-2d-0.1.6
dh_auto_install: make -j1 install DESTDIR=/tmp/binarydeb/ros-kinetic-mrpt-graphslam-2d-0.1.6/debian/ros-kinetic-mrpt-graphslam-2d AM_UPDATE_INFO_DIR=no returned exit code 2
make[1]: *** [override_dh_auto_install] Error 2
```

**Comment by jlblancoc** [comment]
Thanks for reporting, @tfoote .

@bergercookie Hey, I just noticed you self-assigned this one... great! I just pushed a few commits (spread to all mrpt-* ros packages) to fix FTBFS against mrpt-2, just in case you want to git pull before making any changes. 
Note however, that current mrpt ros packages are published for mrpt-1.5...

**Comment by jlblancoc** [comment]
@tfoote : This one is solved, after: 
https://github.com/ros/rosdistro/pull/20906
https://github.com/ros/rosdistro/pull/20907
https://github.com/ros/rosdistro/pull/20908 (this one is just an extra version update)

Thanks
